includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -3060,7 +3066,7 @@ public function get_upe_enabled_payment_method_statuses() {
 			}
 		}
 
-		return 0 === count( $statuses ) ? [
+		return array() === $statuses ? [
```

Comment:
Based on the PHPCS results, it looks like this needs to be `[]` instead of `array()`.
```suggestion
		return [] === $statuses ? [
```